portfolio/src/main/webapp/script.js

```diff
@@ -120,9 +127,34 @@ function drawStatesMap() {
         region: 'US',
         displayMode: 'regions',
         resolution: 'provinces',
+        width: 900,
+        height: 500
     };
 
     var chart = new google.visualization.GeoChart(document.getElementById('states_graph'));
     chart.draw(data, options);
 }
 
+/** Create a chart mapping the population growth rate of the US */
+function drawPopulationChart() {
+    fetch('/pop-data').then(response => response.json()).then((growthRate) => {
+        const data = new google.visualization.DataTable();
+        data.addColumn('string', 'Year');
+        data.addColumn('number', 'Growth-Rate');
+
+        Object.keys(growthRate).forEach((year) => {
+            data.addRow([year, growthRate[year]]);
+        });
```

Comment:
could this just be 
growthRate.forEach((year, growthRate) => data.addRow([year, growthRate]));

Reply:
No, that didn't work when I tried it.